plugins/telink-dfu/fu-telink-dfu-archive.c

```diff
@@ -25,7 +30,11 @@ fu_telink_dfu_archive_load_file(FuTelinkDfuArchive *self,
 				FwupdInstallFlags flags,
 				GError **error)
 {
-	const gchar *bl_type_keys[] = {"beta", "ota-v1"};
+	FuTelinkDfuBlType bl_type_keys[] = {
+	    {"beta", FWUPD_VERSION_FORMAT_TRIPLET},
+	    {"ota-v1", FWUPD_VERSION_FORMAT_TRIPLET},
+	    {"usb-dongle-simple", FWUPD_VERSION_FORMAT_PAIR},
+	};
```

Comment:
you can do this inline if that's helpful:

	struct {
		const gchar *name;
		FwupdVersionFormat ver_format;
	} vendor_names[] = {
		{ "beta", FWUPD_VERSION_FORMAT_TRIPLET },
	};

Reply:
resolved in 7f7093e2519847049ab7f7cadba2f2713c2e13a2
removed the type definition.